test/unit/utils/api-request.spec.ts

```diff
@@ -766,11 +815,7 @@ describe('HttpClient', () => {
         'content-type': 'application/json',
       });
     mockedRequests.push(scope2);
-    const client = new HttpClient({
-      maxRetries: 1,
-      maxDelayInMillis: 1000,
-      statusCodes: [503],
-    });
+    const client = new HttpClient();
```

Comment:
Shouldn't this use `testRetryConfig()` ?

Reply:
Done